docs/development/release.rst

```diff
@@ -21,6 +21,16 @@ The flowchart is:
 In other words, the patch bump should only be used to cherry-pick a subset of
 changes present on the master on a previous version.
 
+Release Checklist
+-----------------
+
+For each release types, apply the following checklist:
+
+* The newsfragment were squash to generate the block added to `HISTORY.rst`.
```

Comment:
"were squashed"

Reply:
This comment could use a suggestion https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/commenting-on-a-pull-request#adding-line-comments-to-a-pull-request

Reply:
done